### DISCUSSION TITLE
<title>hovercards on PR list hide other PRs</title>

### DISCUSSION THREAD

**Discussion by Zhtwn** [community/community#45077] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

The hovercards on the PR list are making it very hard to use. When I am scanning through the list of PRs, the hovercards always come up on top of other listed PRs and make it impossible to see the full list. I end up spending much of my time moving my cursor around so that the hovercards go away and I can acutally read the issue list.

I cannot find any place in the settings to turn off these impediments to productivity.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Zhtwn ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
